tests/cover/test_simple_characters.py

```diff
@@ -90,10 +91,14 @@ def test_whitelisted_characters_alone():
 
 
 def test_whitelisted_characters_overlap_blacklisted_characters():
-    with pytest.raises(InvalidArgument):
+    good_chars = u'te02тест49st'
+    bad_chars = u'ts94тсет'
+    with pytest.raises(InvalidArgument) as exc:
         characters(min_codepoint=ord('0'), max_codepoint=ord('9'),
-                   whitelist_characters=u'te02тест49st',
-                   blacklist_characters=u't').example()
+                   whitelist_characters=good_chars,
+                   blacklist_characters=bad_chars).example()
+        assert good_chars in text_type(exc)
+        assert bad_chars in text_type(exc)
```

Comment:
Rats, this fails on 2.x because `repr()` of a unicode string uses escaping
```
>>> print u'%r' % u'te02тест49st'
u'te02\u0442\u0435\u0441\u044249st'
```